library/common/glfw/embedder.cc

```diff
@@ -98,12 +98,14 @@ static void GLFWFramebufferSizeCallback(GLFWwindow *window, int width_px,
 
   double dpi = state->window_pixels_per_screen_coordinate *
                state->monitor_screen_coordinates_per_inch;
+  double pixel_ratio = dpi / kDpPerInch;
+  if (pixel_ratio < 1) pixel_ratio = 1;
```

Comment:
This can all become:
```
double pixel_ratio = std::max(dpi / kDpPerInch);
```

It would be good to add a comment above that line about why this done (i.e., that scaling down on standard resolution monitors gives undesirable results).